redbot/cogs/mod/mod.py

```diff
@@ -1304,36 +1304,53 @@ def __unload(self):
 
         if not user:
             user = author
+        if isinstance(user, int):
+            user_id = user
+            user = self.bot.get_user(user_id)
+            if not user:
+                try:
+                    user = await self.bot.get_user_info(user_id)
+                except discord.errors.Forbidden:
```

Comment:
`Forbidden` is a 403 error. You want to catch `NotFound`, a 404 error, as per my last comment.